tools/build-rpms.py

```diff
@@ -106,11 +106,22 @@ def create_ec2_instances(cleanup_actions, args, keypair):
             }
         ]
 
+        img = ec2.describe_images(ImageIds=[arch_info[a]["ImageId"]])
         instance = ec2.create_instances(
             ImageId=arch_info[a]["ImageId"],
             MinCount=1,
             MaxCount=1,
             InstanceType=arch_info[a]["InstanceType"],
+            BlockDeviceMappings=[
+                {
+                    "DeviceName": img['Images'][0]['RootDeviceName'],
```

Comment:
It probably is fine to just crash here if the arrays don't match up, right?

Reply:
yea but also every image should have at least 1 block device attached to it by default.

Reply:
Sure, I approved anyway "as it should not happen" but rather if it would - is a stacktrace ok (I guess so…) or a more controlled error